### PR TITLE
Separate json records into multiple messages

### PR DESCRIPTION
**Pull request by eemhu**
If JSON parsing fails, returns original event message as-is

Fixes issue
* #39

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -162,6 +162,17 @@
...
+    <!-- JSON libraries -->
+    <dependency>
+      <groupId>jakarta.json</groupId>
+      <artifactId>jakarta.json-api</artifactId>
+      <version>2.1.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.parsson</groupId>
+      <artifactId>parsson</artifactId>
+      <version>1.1.7</version>
```
[the remark this answers is not in the document]
Comment (eemhu) [reply pom.xml:174]: It is listed https://jakarta.ee/specifications/jsonp/2.1/ here, I think the one you linked is for an earlier version of the specification.

[the remark this answers is not in the document]
Comment (MoonBow-1) [reply pom.xml:174]: They seem to be different projects, but at least Parsson is compatible with the latest specification

**Review by MoonBow-1 — APPROVED** [review]
Looks good to me now

File: src/test/java/com/teragrep/aer_02/SyslogBridgeTest.java [hunk 2]
```diff
@@ -154,6 +155,50 @@ void testSyslogBridge() {
...
+        final String[] seqNums = new String[] {
+                "0", "0", "0", "1", "1", "1", "2", "2", "2"
+        };
+
+        int loops = 0;
+        for (String message : messages) {
+            final RFC5424Frame frame = new RFC5424Frame(false);
+            frame.load(new ByteArrayInputStream(message.getBytes(StandardCharsets.UTF_8)));
+            Assertions.assertTrue(Assertions.assertDoesNotThrow(frame::next));
+            Assertions.assertEquals("\"record" + ((loops % 3) + 1) + "\"", frame.msg.toString());
```
Comment (kortemik) [inline src/test/java/com/teragrep/aer_02/SyslogBridgeTest.java:192]: please make assertions explicit by using a map and remove by key

File: src/main/java/com/teragrep/aer_02/json/JsonRecords.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        final String[] rv = new String[] {
+            event
+        };
+
+        final JsonStructure mainStructure;
+        try (final JsonReader reader = Json.createReader(new StringReader(event))) {
+            mainStructure = reader.read();
+        }
+        catch (JsonParsingException e) {
+            return rv;
```
Comment (kortemik) [inline src/main/java/com/teragrep/aer_02/json/JsonRecords.java:77]: comment that this is a passthrough as-is

**Review by kortemik — CHANGES_REQUESTED** [review]

File: src/test/java/com/teragrep/aer_02/json/JsonRecordsTest.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+    @Test
+    void testNonJsonRecordsCase() {
+        final String records = "{{]///...<>;";
+        JsonRecords jr = new JsonRecords(records);
+        final String[] result = jr.records();
+        Assertions.assertEquals(1, result.length);
+        Assertions.assertEquals("{{]///...<>;", result[0]);
+    }
+
```
[the remark this answers is not in the document]
Comment (eemhu) [reply src/test/java/com/teragrep/aer_02/json/JsonRecordsTest.java:89]: added

**Review by kortemik — APPROVED** [review]

**Pull request merged**